lighty-core/lighty-controller/pom.xml

```diff
@@ -35,6 +35,27 @@
             <groupId>io.lighty.core</groupId>
             <artifactId>lighty-common</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpclient</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>commons-io</groupId>
+            <artifactId>commons-io</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.json</groupId>
+            <artifactId>json</artifactId>
+            <version>20131018</version>
+        </dependency>
+        <dependency>
+            <groupId>io.fabric8</groupId>
+            <artifactId>kubernetes-client</artifactId>
```

Comment:
Oh no no, lighty-controller must not have a hard dependency on K8S. Code needing this should be in  a separate artefact and be wired with lighty-controller properly

Reply:
Extracted to separate artifact